hack/test-end-to-end.sh

```diff
@@ -346,6 +348,20 @@ echo "[INFO] Starting build from ${STI_CONFIG_FILE} and streaming its logs..."
 wait_for_build "test"
 wait_for_app "test"
 
+# Remote command execution
+echo "[INFO] Validating exec"
+frontend_pod=$(oc get pod -l deploymentconfig=frontend -t '{{(index .items 0).metadata.name}}')
+# when running as a restricted pod the registry will run with a pre-allocated
+# user in the neighborhood of 1000000+.  Look for a substring of the pre-allocated uid range
+oc exec -p ${frontend_pod} id | grep 10
+
+# Port forwarding
+echo "[INFO] Validating port-forward"
+oc port-forward -p ${frontend_pod} 10080:8080  &> "${LOG_DIR}/port-forward.log" &
+wait_for_url_timed "http://localhost:10080" "[INFO] Frontend says: " $((10*TIME_SEC))
```

Comment:
> was there something wrong with the healthz check?

The /healthz check is for a registry, not a ruby frontend.  Registry is in default where e2e-user doesn't have rights.  e2e-user checks the ruby front-end.